src/astro/utils/load.py

```diff
@@ -58,14 +62,71 @@ def load_file_into_dataframe(
         elif filetype == FileType.JSON:
             dataframe = pd.read_json(stream, **kwargs)
         elif filetype == FileType.NDJSON:
-            dataframe = pd.read_json(stream, lines=True, **kwargs)
+            dataframe = flatten_ndjson(normalize_config, stream)
         elif filetype == FileType.PARQUET:
             dataframe = pd.read_parquet(stream, **kwargs)
         else:
             raise ValueError(f"Unable to load file '{filepath}' of type '{filetype}'")
         return dataframe
 
 
+def flatten_ndjson(
+    normalize_config: Union[None, dict], stream: io.TextIOWrapper
+) -> pd.DataFrame:
+    """
+    Flatten the nested ndjson/json.
+     :param normalize_config: parameters in dict format of pandas json_normalize() function.
+        https://pandas.pydata.org/docs/reference/api/pandas.json_normalize.html
+     :param stream: io.TextIOWrapper object for the file
+     :type normalize_config: dict
+     :type stream: io.TextIOWrapper
+     :return: return dataframe containing the loaded data
+     :rtype: `pandas.DataFrame`
+    """
```

Comment:
```suggestion
    """
    """
    Flatten the nested ndjson/json.

    :param normalize_config: parameters in dict format of pandas json_normalize() function.
        https://pandas.pydata.org/docs/reference/api/pandas.json_normalize.html
    :param stream: io.TextIOWrapper object for the file
    :type normalize_config: dict
    :type stream: io.TextIOWrapper
    :return: return dataframe containing the loaded data
    :rtype: `pandas.DataFrame`
    """
```

You can also ignore type since you are using Type hints